en/guide/mavlink_version.md

```diff
@@ -1,27 +1,52 @@
 # MAVLink Versions
 
-MAVLink is deployed in two major versions: *MAVLink 1.0* and [MAVLink 2.0](../guide/mavlink_2.md).
+MAVLink has been deployed in a number of versions:
+- [MAVLink 2.0](../guide/mavlink_2.md): Current/recommended major version. Adopted by major users early 2017. 
+- *MAVLink v1.0*: Widely adopted around 2013. Still used by a number of legacy peripherals.
+
+The *MAVLink 2.0* C/C++ and Python libraries are backwards compatible with MAVLink 1.0 (support both protocols).
+[Version Handshaking](#version_handshaking) and [Negotiating Versions](#negotiating_versions) explain how to choose which version is used.
+  
+> **Note** *MAVLink v0.9* is a pre-release version that is no longer supported. 
+  The associated message set deleted in August 2018. 
+  Legacy code may be present in generator and test code. 
+
+
+## Determining Protocol/Message Version
+
+A library's MAVLink support can be determined in a number of ways:
+* [AUTOPILOT_VERSION](../messages/common.md#AUTOPILOT_VERSION)`.capabilities` can be checked against the [MAV_PROTOCOL_CAPABILITY_MAVLINK2](../messages/common.md#MAV_PROTOCOL_CAPABILITY_MAVLINK2) flag to verify MAVLink 2 support.
```

Comment:
Shouldn't there be a `MAV_PROTOCOL_CAPABILITY_MAVLINK1`? Otherwise how can a system that just supports MAVLink 2 indicate that it doesn't support MAVLink1?

Reply:
Right now the protocol expects that you always support v1. Can you file an issue against mavlink/mavlink for that?